binds/python/bind_vasculature.cpp

```diff
@@ -36,6 +36,21 @@ void bind_vasculature(py::module& m) {
              "throw RawDataError if the id is out of range",
              "section_id"_a)
 
+
+        .def_property_readonly(
+            "section_offsets",
+            [](morphio::vasculature::Vasculature& vasculature) {
+                return as_pyarray(vasculature.sectionOffsets());
+            },
+            "Returns a list with offsets to access data of a specific section in the points\n"
+            "and diameters arrays.\n"
+            "\n"
+            "Example: accessing diameters of n'th section will be located in the DIAMETERS\n"
+            "array from DIAMETERS[sectionOffsets(n)] to DIAMETERS[sectionOffsets(n+1)-1]\n"
+            "\n"
+            "Note: for convenience, the last point of this array is the points array size\n"
```

Comment:
not sure I follow; I think the word `point` in `the last point of this array` is confusing, but I'm still not following when I substitute `element`

Reply:
Let's say we have the following points (I list their ids):
`[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]`

And three sections:
```
S1: [0, 1, 2]
S2: [3, 4, 5]
S3: [6, 7, 8, 9]
```
The offsets will be:
`[0, 3, 6, 10]`

So the last entry is equal to the size of the points dataset. This allows for applying a diff for instance to calculate the section length (in terms of points) for each section without having to allocate an extra array to add that extra value.

Reply:
ah, ok; basicaly if that example can be copied to the test in `tests/test_10_vasculature.py` that would be good

Reply:
sure, I copied it there.